### PR TITLE
worked on defining an optional parameter to specify whether to load i…

### PR DESCRIPTION
**Pull request by Innovative-GauravKochar**
This PR is to pass an optional paramter in loadIframe function which will specify whether iframe should get load in head or body.
**Testing in Staging**
Screenshot 2023-04-17 at 1 53 00 PM

### CODE REVIEW

File: lib/protos.js [hunk 1]
```diff
@@ -231,9 +231,10 @@ exports.reset = function() {
  * (e.g. `userId` in '<img src="https://whatever.com/{{ userId }}">').
  * @param {Function} [callback=noop] A callback, invoked when the tag finishes
  * loading.
+ * @param {Boolean} moveToBody to specify whether to load the iframe in head or body
  */
 
-exports.load = function(name, locals, callback) {
+exports.load = function(name, locals, callback, moveToBody) {
```
Comment (pooyaj) [inline lib/protos.js:237]:
```suggestion
exports.load = function(name, locals, callback, shouldMoveIframeToBody) {
```

File: lib/protos.js [hunk 2]
```diff
@@ -248,6 +249,7 @@ exports.load = function(name, locals, callback) {
   if (!template) throw new Error(fmt('template "%s" not defined.', name));
   var attrs = render(template, locals);
   callback = callback || noop;
+  moveToBody= moveToBody || false;
```
Comment (pooyaj) [inline lib/protos.js:252]:
```suggestion
  moveToBody = moveToBody || false;
```

File: lib/protos.js [hunk 3]
```diff
@@ -269,7 +271,7 @@ exports.load = function(name, locals, callback) {
     }, attrs);
     break;
   case 'iframe':
-    el = loadIframe(attrs, callback);
+    el = loadIframe(attrs,callback,moveToBody);
```
Comment (pooyaj) [inline lib/protos.js:274]:
```suggestion
    el = loadIframe(attrs, callback, moveToBody);
```

**Review by pooyaj — APPROVED** [review]
small nit-pick/formatting. otherwise lgtm

File: lib/protos.js (hunk 1 above)
Comment (pooyaj) [reply lib/protos.js:237]: renaming the parameter here to make it clear it only impacts iframe, but not other types of elements supported in this function.

File: lib/protos.js [hunk 4]
```diff
@@ -248,6 +249,7 @@ exports.load = function(name, locals, callback) {
   if (!template) throw new Error(fmt('template "%s" not defined.', name));
   var attrs = render(template, locals);
   callback = callback || noop;
+  shouldMoveIframeToBody = shouldMoveIframeToBody || false;
```
Comment (rhall-twilio) [inline lib/protos.js:252]: :nit: you can do function(name, locals, callback, shouldMoveIframeToBody = false)

Same for the other PR.

**Pull request merged**